include/dmtcp.h

```diff
@@ -182,13 +182,13 @@ EXTERNC const char* dmtcp_get_tmpdir(void);
 
 EXTERNC const char* dmtcp_get_ckpt_dir(void) __attribute ((weak));
 #define dmtcp_get_ckpt_dir() \
- (dmtcp_get_ckpt_dir ? dmtcp_get_ckpt_dir() : NULL)
+ (dmtcp_get_ckpt_dir ? dmtcp_get_ckpt_dir() : "")
```

Comment:
Other users may be testing `dmtcp_get_ckpt_dir` for the NULL value. If we cast the function call to return `(char *)` (i.e., drop the const), then the else branch can again return NULL instead of the empty string.